src/averageAges.js

```diff
@@ -20,6 +20,21 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mens = people
+    .filter(person => {
+      const personCenturyDeath = Math.ceil(person.died / 100);
+      const personSex = person.sex;
+
+      return century
+        ? personSex === 'm' && century === personCenturyDeath
+        : personSex === 'm';
+    });
+
+  const averageMenAge = mens
+    .reduce((prev, { born, died }) => prev + (died - born), 0) / mens.length
```

Comment:
You do the same calculations with reduce in every function. Maybe we should move this calculation to a separate helper?